cpp/dcgan/dcgan.cpp

```diff
@@ -153,13 +153,13 @@ int main(int argc, const char* argv[]) {
           torch::binary_cross_entropy(fake_output, fake_labels);
       g_loss.backward();
       generator_optimizer.step();
-
+      batch_index++
```

Comment:
missing semicolon at the end

Reply:
Oops, sorry i didn't catch that before putting in the PR. I made the change and can confirm that the new code runs and outputs are similar to what's on the READM.md here 

https://github.com/pytorch/examples/blob/master/cpp/dcgan/README.md